ibis/expr/types/relations.py

```diff
@@ -225,6 +225,26 @@ def cast(self, schema: SupportsSchema) -> Table:
             ...
         ibis.common.exceptions.IbisError: Cast schema has fields that are not in the table: ['foo']
         """
+        return self._cast(schema, strict=True)
+
+    def try_cast(self, schema: SupportsSchema, strict=True) -> Table:
+        """Cast the columns of a table.
+        If the cast fails for a row, the value is returned
```

Comment:
I think for proper formatting you'll want a newline here.

```suggestion

        If the cast fails for a row, the value is returned
```

Reply:
done